uast/diff/diff.go

```diff
@@ -0,0 +1,283 @@
+package diff
+
+import (
+	// TODO: integrate with https://github.com/src-d/lapjv if perf is unacceptable
```

Comment:
This TODO won't happen until `src-d/lapjv` is rewritten in pure Go. We won't bind to C in SDK.

Reply:
I wasn't sure whether "won't" or "don't really want to" so left the reminder our impl exists